### DISCUSSION TITLE
Customized VSCodium build

### DISCUSSION THREAD

**Discussion by AdrianG-OSS** [VSCodium/vscodium#711] [Q&A]
Hi everyone

We are a company that want to develop a tool that would basically be an Editor plus a couple of our extensions on top. We would like to base our work on VSCodium that looks to offer us a solid foundation. We would however need to do further tweaking to the code, like the removal of the marketplace/Debug from the UI, removal of some bundled extensions plus of some menu items. How difficult would this be to do? Any info/tips will be much appreciated. Thank you!

**Comment by Tafnab** [comment]
Looks like nobody's home.

**Comment by daiyam** [comment]
Well, you could take a look at my rebuild (https://github.com/zokugun/MrCode) but I haven't updated it in a while and I've moved some of the patchs/fixes directly into VSCodium...

**Reply by Tafnab** [reply]
Will do. I'm just trying to get it to stop checking the damned store every time it finds a new file type. Does yours do that?

**Reply by daiyam** [reply]
There is a setting for that: `"extensions.ignoreRecommendations": true`. Only works on restart.

**Comment by xiaochao00** [comment]
@AdrianG-OSS How did you build vscodium and develop your software, I have the same idea with you; Can you share me some methods?
